packages/cdk/resources/StaticContentBucket.ts

```diff
@@ -0,0 +1,116 @@
+import {Fn, RemovalPolicy} from "aws-cdk-lib"
+import {Role} from "aws-cdk-lib/aws-iam"
+import {CfnKey, Key} from "aws-cdk-lib/aws-kms"
+import {
+  BlockPublicAccess,
+  Bucket,
+  BucketAccessControl,
+  BucketEncryption,
+  CfnBucket,
+  CfnBucketPolicy,
+  ObjectOwnership
+} from "aws-cdk-lib/aws-s3"
+import {Construct} from "constructs"
+
+import {AllowCloudfrontGetObjectPolicyStatement} from "../policies/s3/AllowCloudfrontGetObjectPolicyStatement"
+import {AllowCloudfrontKmsKeyAccessPolicy} from "../policies/kms/AllowCloudfrontKmsKeyAccessPolicy"
+
+/**
+ * Resources for a static content S3 bucket
+
+ */
+
+export class StaticContentBucket extends Construct{
+  public readonly bucket: Bucket
+  public kmsKey: Key
+
+  public constructor(scope: Construct, id: string){
+    super(scope, id)
+
+    // Context
+    /* context values passed as --context cli arguments are passed as strings so coerce them to expected types*/
+    const allowAutoDeleteObjects: boolean = this.node.tryGetContext("allowAutoDeleteObjects") === "true"
+    const cloudfrontDistributionId: string = this.node.tryGetContext("cloudfrontDistributionId")
+
+    // Imports
+    const auditLoggingBucket = Bucket.fromBucketArn(
+      this, "AuditLoggingBucket", Fn.importValue("account-resources:AuditLoggingBucket"))
+
+    const deploymentRole = Role.fromRoleArn(
+      this, "deploymentRole", Fn.importValue("ci-resources:CloudFormationDeployRole"))
+
+    // Resources
+    const kmsKey = new Key(this, "KmsKey", {
+      enableKeyRotation: true,
+      removalPolicy: RemovalPolicy.DESTROY
+    })
+    kmsKey.addAlias("alias/StaticContentBucketKmsKey")
+
+    const bucket = new Bucket(this, "Bucket", {
+      encryption: BucketEncryption.KMS,
+      bucketKeyEnabled: true,
+      encryptionKey: kmsKey,
+      enforceSSL: true,
+      blockPublicAccess: BlockPublicAccess.BLOCK_ALL,
+      accessControl: BucketAccessControl.PRIVATE,
+      objectOwnership: ObjectOwnership.BUCKET_OWNER_ENFORCED,
+      serverAccessLogsBucket: auditLoggingBucket,
+      serverAccessLogsPrefix: "/static-content",
+      removalPolicy: RemovalPolicy.DESTROY,
+      autoDeleteObjects: allowAutoDeleteObjects // if true forces a deletion even if bucket is not empty
+    })
+
+    bucket.grantReadWrite(deploymentRole)
+
+    /* As you cannot modify imported policies, cdk cannot not update the s3 bucket with the correct permissions
+    for OAC when the distribution and bucket are in different stacks
+    !! This can only be added after the distribution has been deployed !! */
+    if (cloudfrontDistributionId){
```

Comment:
this needs to be 
```
    if (cloudfrontDistributionId !== "") {
```
as it will be set to empty string in .github/workflows/cdk_release_code.yml when you do the jq replace if `aws cloudformation list-exports` returns nothing